packages/react-slider/src/components/Slider/Slider.test.tsx

```diff
@@ -164,7 +138,6 @@ describe('Slider', () => {
     fireEvent.pointerDown(sliderRoot, { clientX: 0, clientY: 0 });
 
     expect(onChange).toHaveBeenCalledTimes(1);
-    expect(onChange.mock.calls[0][0]).toEqual(0);
```

Comment:
You could change this to `expect(onChange.mock.calls[0][1]).toEqual({ value: 0 })` and it would still work (similar elsewhere)